cli/mender-artifact/modify.go

```diff
@@ -81,8 +76,12 @@ func modifyArtifact(c *cli.Context) error {
 
 	if isArtifact {
 		// re-create the artifact
-		err := repackArtifact(comp, c.Args().First(), modifyCandidates[0].path,
-			c.String("key"), c.String("name"))
+		rootfs := ""
+		if modifyCandidates != nil {
```

Comment:
What happens if there are no partitions to repack? (ie, modifyCandidates is nil? )

Reply:
It means that we haven't made any change in the partitions, only in the headers of the Artifacts, so no repacking is needed.